Short-Answer/Algorithms_Answers.md

```diff
@@ -12,4 +12,6 @@ c) O(log n), recursive
 
 ## Exercise II
 
+divide n by 2 to get a possible midpoint, check if answer is to the right or left. discard either side if value is not on that side. repeat until the right floor is found. also we want to check the right side first according to the question
```

Comment:
This is not incorrect but it would be helpful to try to be a little bit more explicit with your explanation. i.e. I would approach this using a binary search method. Starting at the middle floor, drop the first egg. If it breaks...and so on.